Solutions/MCBSTM32F400/netmfdbg/netmfdbg.uvoptx

```diff
@@ -1078,6 +1281,260 @@
     </TargetOption>
   </Target>
 
+  <Target>
```

Comment:
No it's driven by uVision configuration to the project to debug the "nonet" project as welll